app/create/components/StepThree.tsx

```diff
@@ -17,6 +17,9 @@ const StepThree = ({
   ) => void;
   address: string | undefined;
 }) => {
+
+  const timezones = ["GMT", "UTC"];
```

Comment:
It should be GMT or local time. Check out the unix time converter function in the in the helper folder `index.ts`

Reply:
please resolve merge conflicts